### PR TITLE
Add some additional text about getting help.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -68,6 +68,32 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
 
 The Kubernetes mailing list is google-containers@googlegroups.com
 
+# Getting help
+If your problem isn't answered by any of guides above, there are variety of ways for you to get help from the Kubernetes team.
```
Comment (davidopp) [inline docs/troubleshooting.md]: any of -> any of the

**Comment by davidopp** [comment]
Please also update the top-level README.md, which covers some of these same things
(see section "Community, discussion and support")

File: docs/troubleshooting.md (hunk 1 above)
Comment (brendandburns) [inline docs/troubleshooting.md]: done.

**Comment by brendandburns** [comment]
Comments addressed, ptal.

Thanks!
--brendan

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -68,6 +68,37 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
 
 The Kubernetes mailing list is google-containers@googlegroups.com
 
+# Getting help
+
+If your problem isn't answered by any of guides above, there are variety of ways for you to get help from the Kubernetes team.
```
Comment (davidopp) [inline docs/troubleshooting.md]: any of guides -> any of the guides

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -68,6 +68,37 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
...
+
+# Help! My question isn't covered!  I need help now!
+
+## IRC
+
+The Kubernetes team hangs out on IRC at [```#google-containers```](https://botbot.me/freenode/google-containers/) on freenode.  Feel free to come and ask any and all questions there.
+
+## Mailing List
+
+The Kubernetes mailing list is google-containers@googlegroups.com
```
Comment (davidopp) [inline docs/troubleshooting.md]: maybe write this as
`[google-containers@googlegroups.com](https://groups.google.com/forum/#!forum/google-containers)

Comment (davidopp) [inline docs/troubleshooting.md]: sorry, that should say
`[google-containers@googlegroups.com](https://groups.google.com/forum/#!forum/google-containers)`

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -68,6 +68,37 @@ The Kubernetes team hangs out on IRC at [`#google-containers`](https://botbot.me
 
 The Kubernetes mailing list is google-containers@googlegroups.com
 
+# Getting help
```
Comment (thockin) [inline docs/troubleshooting.md]: Why is this pgph in the doc twice?

File: docs/troubleshooting.md (hunk 2 above)
Comment (davidopp) [inline docs/troubleshooting.md]: Actually, it looks like the doc got a bit screwed up. Some of the sections are duplicated, like "Getting help" and "Help! My question isn't covered! I need help now!"

**Comment by brendandburns** [comment]
Fixed.  I think I must have ended up with slightly different commits in two different branches, and somehow things got borked...

Sorry, this is fixed now.

ptal

Thanks
--brendan

**Comment by davidopp** [comment]
OK, looking now

**Comment by davidopp** [comment]
LGTM

**Comment by krousey** [comment]
Shippable failed because #11595 moved some files and paths weren't updated in the examples test. @erictune is working on that in a separate PR, but I won't hold up this one because of it.

**Pull request merged**